packages/blade/src/components/Amount/__tests__/__snapshots__/Amount.web.test.tsx.snap

```diff
@@ -1934,36 +1934,6 @@ exports[`<Amount /> should render isStrikethrough={true} 1`] = `
   opacity: 0.64;
 }
 
-@media screen and (min-width:320px) {
-  .c5.c5.c5.c5.c5 {
-    border-bottom-style: solid;
-  }
-}
```

Comment:
Did we change anything in Amount or Box?

Reply:
yup we changed it in Box. We were adding borderBottomStyle by default earlier which was making it add it to all media queries.

Now we only add it to given breakpoint so it removed the additional code from media queries